packages/domain-picker/src/hooks/use-domain-suggestions.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { DomainSuggestions } from '@automattic/data-stores';
+
+import { DOMAIN_SUGGESTION_VENDOR, PAID_DOMAINS_TO_SHOW } from './constants';
+
+const DOMAIN_SUGGESTIONS_STORE = DomainSuggestions.register( {
+	vendor: DOMAIN_SUGGESTION_VENDOR,
+} );
```

Comment:
according to https://github.com/Automattic/wp-calypso/pull/43045#discussion_r436576443 we should register/configure the store here but let the application (Gutenboarding/FSE) do this and assume it has been already registered when accessing it directly via the store key inside the shared package.